api/tests/Feature/CommunityInterestTest.php

```diff
@@ -331,6 +333,12 @@ public function testCommunityInterestsPaginatedRoles(): void
             [],
         )->assertJsonFragment(['total' => 1])
             ->assertJsonFragment(['id' => $communityInterestModel->id]);
+        // base user can see their own community interest
+        $this->actingAs($owningUser, 'api')->graphQL(
```

Comment:
Intelephense is showing me an error for this but it seems to work. Any suggestions?

Reply:
You can type hint to help it understand that it's a single user coming back, not a collection of `Model`s.
![image](https://github.com/user-attachments/assets/2bbb1360-001a-4eeb-8048-c00881fd8c9f)